optima/results.py

```diff
@@ -100,6 +101,46 @@ def __repr__(self):
         output += '============================================================\n'
         output += objrepr(self)
         return output
+        
+    
+    def __add__(self, R2):
+        ''' Define how to add two Resultsets '''
+        if type(R2)!=Resultset: raise OptimaException('Can only add two results sets')
```

Comment:
the __add__ method operates on two objects but can be done in succession, e.g. R1+R2+R3 = (R1+R2)+R3 etc.